### PR TITLE
Edit 'Develop Using a Domain Without Changing DNS'

### PR DESCRIPTION
**Pull request by aeligature**
add dig command

### CODE REVIEW

File: source/docs/articles/sites/domains.md [hunk 1]
```diff
@@ -33,12 +33,29 @@ You can simultaneously add both the bare domain name and the www subdomain. This
 ### Develop Using a Domain Without Changing DNS
 Sometimes it's useful to develop on a site using a specific domain, but the overhead of temporarily changing DNS is too much. Use the following workaround to allow your local workstation to access your Pantheon site by the desired domain without changing DNS. This requires a paid plan.
 
-1. From the Pantheon Dashboard, add the domain to the target site environment.
-2. Add a line to your local <a href="https://en.wikipedia.org/wiki/Hosts_(file)">Hosts file</a> with the provided IP address, default value for the Live environment, and the bare domain.
+<div class="alert alert-danger" role="alert">
+<h4>Important</h4>This process is for short-term DNS testing. Endpoint IPs will change over time. Be sure to use the Dashboard's recommendations and provided IPs when going Live.</div>
```
Comment (ari-gold) [inline source/docs/articles/sites/domains.md]: Delete "...and provided IPs.."

File: source/docs/articles/sites/domains.md [hunk 2]
```diff
@@ -33,12 +33,29 @@ You can simultaneously add both the bare domain name and the www subdomain. This
...
+2. From the Pantheon Dashboard, add your domain to the target site environment.
+3. Add a line to your local <a href="https://en.wikipedia.org/wiki/Hosts_(file)">Hosts file</a> with one of the IP addresses revealed via dig, default value for the Live environment, and the bare domain.
 
 Example:
 ```
-192.237.142.203 live-example.pantheon.io example.com
+192.123.456.789 live-yoursite.pantheon.io yoursite.com
 ```
 
 Remove this entry when you're ready to configure DNS.
```
Comment (ari-gold) [inline source/docs/articles/sites/domains.md]: Add note to follow Site Dashboard recommendations when going Live, just to emphasize that these test settings should not be used to configure DNS?

Also wondering if it'd be helpful to make the "Remove this entry..." step 4.

**Comment by nataliejeremy** [comment]
Thanks for the update @aeligature 
I incorporated your feedback @ari-gold Please review and +1

File: source/docs/articles/sites/domains.md [hunk 3]
```diff
@@ -33,15 +33,30 @@ You can simultaneously add both the bare domain name and the www subdomain. This
 ### Develop Using a Domain Without Changing DNS
 Sometimes it's useful to develop on a site using a specific domain, but the overhead of temporarily changing DNS is too much. Use the following workaround to allow your local workstation to access your Pantheon site by the desired domain without changing DNS. This requires a paid plan.
 
-1. From the Pantheon Dashboard, add the domain to the target site environment.
-2. Add a line to your local <a href="https://en.wikipedia.org/wiki/Hosts_(file)">Hosts file</a> with the provided IP address, default value for the Live environment, and the bare domain.
+<div class="alert alert-danger" role="alert">
+<h4>Important</h4>This process is for short-term DNS testing. Endpoint IPs will change over time. Be sure to use the Dashboard's recommendations when going Live.</div>
```
Comment (ari-gold) [inline source/docs/articles/sites/domains.md]: @aeligature @nataliejeremy  Looking at this again...how about a stronger warning that mentions downtime? Maybe:

This process is for short-term testing only. Be sure to use the Dashboard's recommendations when going live, or you will experience downtime when IP addresses change.

**Comment by rachelwhitton** [comment]
+1 Thanks for the pr @aeligature!

**Pull request merged**